pydm/main_window.py

```diff
@@ -327,6 +340,10 @@ def load_tool(self, checked):
 
     @pyqtSlot(bool)
     def reload_display(self, checked):
+        try:
+            curr_file = self.current_file()
+        except IndexError:
```

Comment:
Log message here as well. This should probably be high level so that a user trying to press reload without a display button realizes what has happened.

Reply:
Addressed on the latest commit.

Reply:
You sure it should just be `info`? Calling `reload_display` without a display seems like an `error`

Reply:
I was interpreting it as a minor error, not something critical but your recommendation is valid so I addressed this at the latest commit and changed to `error`.